Ghidra/Framework/Docking/src/main/java/docking/widgets/autocomplete/TextFieldAutocompleter.java

```diff
@@ -70,21 +70,20 @@
  * The simplest use case is to create a text field, create an autocompleter with a custom model,
  * and then attach and show.
  * 
- * {@code
+ *
  * <pre>
  * JTextField field = new JTextField();
  * 
- * AutocompletionModel<String> model = new AutocompletionModel<String>() {
- *     @Override
- *     public Collection<String> computeCompletions(String text) {
+ * {@code AutocompletionModel<String> model = new AutocompletionModel<String>() }{
+ *     &#64;Override
+ *     {@code public Collection<String> computeCompletions(String text)} {
  *         ... // Populate the completion list based on the given prefix.
  *     }
  * }
- * TextFieldAutocompleter<String> completer = new TextFieldAutocompleter<String>(model);
+ * {@code TextFieldAutocompleter<String> completer = new TextFieldAutocompleter<String>(model);
  * completer.attachTo(field);
  * ... // Add the field to, e.g., a dialog, and show.
- * </pre>
- * }
+ * }</pre>
```

Comment:
This was a pain. Not only can `@` not be used within a `{@code}` or `{@literal}` it cannot be escaped either since it isn't being interpreted as html. From there the `}` from the struct was getting in the way.